deepchecks/vision/checks/distribution/image_property_drift.py

```diff
@@ -171,17 +195,73 @@ def compute(self, context: Context) -> CheckResult:
                        f'for the distribution of the following image properties: {properties}.' \
                        '</span>'
 
-            displays = [headnote] + [figures[col] for col in columns_order]
-        else:
-            drifts = None
-            displays = []
+            train_samples = df_train.sample(10)
+            test_samples = df_test.sample(10)
+            train_thumbnail_images, *_ = context.train.sample(*list(train_samples.index))
+            test_thumbnail_images, *_ = context.test.sample(*list(train_samples.index))
+
+            thumbnails = self._prepare_thumbnails_block(
```

Comment:
From the issue description:

> After the graphs, add examples (from user's images) of different images on different scales (5 images exemplifying brightness, then 5 exemplifying aspect ratio etc.)

So per property show examples of images with different property values. Can either do it from train and then from test, or mix train and test but then tell us which is which.

Reply:
I implemented it like this

>Other option - one table of ~10 images and below them all the relevant values per property (takes up less space, but will look worse probably)